upgrade-1.x-to-2.x.sh

```diff
@@ -492,7 +492,7 @@ case $root_part in
                 ;;
         esac
         log "Rebooting..."
-        progress 15 "ResinOS: rebooting to continue..."
+        progress 15 "Rebooting from switched partition"
```

Comment:
maybe "second" or "new" would be clearer than "switched?" it's not clear which partition is meant by "switched"

Reply:
Neither (the pr or the alternatives) feel really good... will keep thinking.